smartprom.py

```diff
@@ -172,7 +173,7 @@ def main():
     """
     starts a server at port 9902 and exposes the metrics
     """
-    start_http_server(9902)
+    start_http_server(int(os.environ.get('PORT', 9902)))
```

Comment:
I would prefer to have an environment variable that clarifies what PORT is being set here. Something like `SMARTCTL_EXPORTER_PORT` for example.